packages/web-api-client-library/src/a11y-service-client.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { injectable } from 'inversify';
+import * as request from 'request-promise';
+
+@injectable()
+export class A11yServiceClient {
+    constructor(private readonly baseUrl: string, private readonly apiVersion = '1.0', private readonly httpRequest = request) {}
```

Comment:
how do you make sure that you don't create more than one http agent?

Reply:
We are using node js request package which provides option to use [forever-agent](https://github.com/request/forever-agent).

Reply:
does it create a new forever agent for every request, or does it reuse the same agent?

Reply:
discussed offline

Reply:
Verified that the client will reuse the same https agent.